src/platform/site-wide/announcements/config/index.js

```diff
@@ -28,6 +29,12 @@ const config = {
       paths: /^\/$/,
       component: WelcomeToNewVAModal,
     },
+    {
+      name: 'pittsburgh-vamc',
+      paths: /^\/pittsburgh-health-care\/(.)*$/,
+      component: VAMCWelcomeModal,
+      region: 'Pittsburgh',
```

Comment:
LGTM as long as you think simply passing a region to the component will give us enough flexibility. But it would be easy to change this down the line if needs change.